docs/cli/coder_users_suspend.md

```diff
@@ -0,0 +1,45 @@
+## coder users suspend
+
+Update a user's status to 'suspended'. A suspended user cannot log into the platform
+
+```
+coder users suspend <username|user_id> [flags]
+```
+
+### Examples
+
+```
+ [48;2;235;229;236m [0m[38;2;255;70;113;48;2;235;229;236m$ coder users suspend example_user[0m[48;2;235;229;236m [0m
```

Comment:
I believe this will format _really_ weird.

Reply:
Yes, looking at what we can do to solve this issue. Looks like it is an issue in the CLI library https://github.com/spf13/cobra/issues/1878